scripts/zpool

```diff
@@ -130,6 +137,15 @@ zpool_start() {
         sleep 1
     done
 
+    if [ -n "$OCF_RESKEY_post_import_cmd" ]; then
+        ocf_run $OCF_RESKEY_post_import_cmd
```

Comment:
Do we maybe want to pass the $OCF_RESKEY_pool as a command line parameter to the script?  The script is going to be called after each pool is imported, so it might be nice to tell which pool it is being called after.

Reply:
Oops, yes.  I'd intended to do that but it got lost.  Thanks for catching it.